cico_run_coverage.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+. cico_setup.sh
+
+COVERAGE="f8osoproxy-coverage"
+PACKAGE_NAME="github.com/containous/traefik"
+
+GOPATH_IN_CONTAINER=/tmp/go
+PACKAGE_PATH=$GOPATH_IN_CONTAINER/src/$PACKAGE_NAME
+
+docker build -t "$COVERAGE" -f Dockerfile.builder .
```

Comment:
Currently we install go 1.9.2 in Dockerfile.builder: https://github.com/fabric8-services/fabric8-oso-proxy/blob/master/Dockerfile.builder

But it's better to use go 1.10 for test coverage collection because it's much faster (in some our repos it's +10x faster) thanks to test cache in go 1.10. But we still need to build our bits which we are deploying using go 1.9.

In other repos (like WIT and Auth) we set an env var USE_GO_VERSION_FROM_WEBSITE to switch code coverage to go 1.10.
Take a look at WIT for example:
https://github.com/fabric8-services/fabric8-wit/blob/master/cico_run_coverage.sh#L5
https://github.com/fabric8-services/fabric8-wit/blob/master/Dockerfile.builder#L5

It's important because without the test cache introduced in go 1.10 the code coverage job often takes too long and killed by Jenkins.

Reply:
@alexeykazakov traefik support Go1.10 [ref](https://github.com/containous/traefik/blob/v1.6/CONTRIBUTING.md#setting-up-your-go-environment) and in discussion with @aslakknutsen we decided to use Go1.10 for oso_proxy and rather having two cico_job (test, coverage) we will include coverage thing in existing cico_run_tests job.

Reply:
I'm fine with adding test coverage to the existing cico_run_tests job but we still use Go 1.9.2 to build the binaries - https://github.com/fabric8-services/fabric8-oso-proxy/blob/master/Dockerfile.builder#L23 which can be too slow. We need to switch it to 1.10 then.